src/main/java/com/example/TournamentProject/CSVExportService.java

```diff
@@ -0,0 +1,30 @@
+package com.example.TournamentProject;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.*;
+
+@Service
+public class CSVExportService {
+    @Autowired
+    private TournamentManagement tournamentManagement;
+
+    public void saveInTXTFile() {
+        String tournamentInTXTFile = tournamentManagement.createTournamentName() + ".txt";
+
+        System.out.println("If you want save results enter 'Save Tournament'");
+
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+            try (BufferedWriter writer = new BufferedWriter(new FileWriter(tournamentInTXTFile))) {
```

Comment:
why you`ve used 2 try with resources construction?
try-with-resources With Multiple Resources -> 

```
try (Scanner scanner = new Scanner(new File("testRead.txt"));
    PrintWriter writer = new PrintWriter(new File("testWrite.txt"))) {
    while (scanner.hasNext()) {
	writer.print(scanner.nextLine());
    }
}
```